BeeJet.Bot/Commands/Handlers/Steam/SyncSteamCommandHandler.cs

```diff
@@ -15,29 +15,28 @@ public SyncSteamCommandHandler(Services.SteamAPIService steamAPI)
         }
 
         [BeeJetBotSlashCommand("sync-steam", "Sync steam library with channels", nameof(RegisterOptions))]
-        public async Task SlashCommandExecuted(SlashCommandContext context)
+        public async Task SlashCommandExecuted()
         {
-            if (ulong.TryParse((string)context.DiscordContext.Data.Options.First().Value, out ulong steamId))
+            if (!ulong.TryParse((string)Context.SlashCommandInteraction.Data.Options.First().Value, out ulong steamId))
             {
-                var games = await _steamAPI.GetGamesFromSteamUser(steamId);
-                var gamesWithChannel = context.Guild.Channels.OfType<SocketTextChannel>().Where(b => games.Any(c => c.Equals(b.Name, StringComparison.OrdinalIgnoreCase)));
-                gamesWithChannel = gamesWithChannel.Where(b => !b.Users.Any(u => u.Id == context.DiscordContext.User.Id));
-                if (!gamesWithChannel.Any())
-                {
-                    await context.DiscordContext.RespondAsync("No channels to join");
-                    return;
-                }
-                var builder = new ComponentBuilder();
-                foreach (var game in gamesWithChannel)
-                {
-                    builder.WithButton(game.Name + $"({game.Category.Name})", "join-game-" + game.Name.Replace(" ", "-") + "-------" + game.Category.Name.Replace(" ", "-"));
-                }
-                await context.DiscordContext.RespondAsync("Which channels do you want to join?", ephemeral: true, components: builder.Build());
+                await Context.SlashCommandInteraction.RespondAsync("Not a valid steamid", ephemeral: true);
+                return;
             }
-            else
+
+            var games = await _steamAPI.GetGamesFromSteamUser(steamId);
+            var gamesWithChannel = (await Context.Guild.GetChannelsAsync()).OfType<SocketTextChannel>().Where(b => games.Any(c => c.Equals(b.Name, StringComparison.OrdinalIgnoreCase)));
```

Comment:
Wat is b en c :)

Reply:
Ik heb het aangepast, maar vind het persoonlijk niet mooier erop worden. Wordt super verbose. 
Iedereen mag ik hopen snapt inmiddels linq, als je user.Where(u=> doet, dan lijkt mij dat duidelijk genoeg.
Nu worden die regels super lang, zonder dat het er duidelijker op word, maakt de code mijn inziens minder leesbaar ipv meer leesbaar.

`  gamesWithChannel = gamesWithChannel.Where(discordChannel => !discordChannel.Users.Any(user => user.Id == Context.SlashCommandInteraction.User.Id));
`

@MauricePonte of @pyrocumulus  wat vinden jullie?

Reply:
Ik doe zelf denk ik meestal wel voluitschrijven :) Als de regels dan super lang worden, ga ik de linq query line-wrappen met indenting en dan valt het meestal wel mee. Is het dan nog steeds een rommeltje, dan doe ik de Where() predicate afsplitsen naar een losse methode

Reply:
Ok, had al de commit gedaan om het helemaal uit te schrijven, zal ik dat voortaan ook doen.